src/Core/src/Platform/iOS/MauiTimePicker.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using Foundation;
+using UIKit;
+using RectangleF = CoreGraphics.CGRect;
+
+namespace Microsoft.Maui
+{
+	public class NoCaretField : UITextField
```

Comment:
Maybe move this into a new file as I see the Pickers PR also implements this type. The the first to be merged can be a winner and the rest have to remove .